CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.2.17.0.2
+- Banner impressions are now tracked on ad show using the AD_VIEWABLE method.
```

Comment:
nit:
- Banner impressions are now tracked on ad show using ImpressionTrackingMethod.AD_VIEWABLE.

This removes the ambiguity of the word "method"

Reply:
Thoughts @chauduyphanvu? I can make this change with a 👍🏼

Reply:
Sure go for it